lib/model/eval.js

```diff
@@ -3,6 +3,19 @@ const preferenceManager = require('../utils/preference-manager');
 
 const { logger } = require('../utils/logger');
 
+const handlerMiddleware = (socket) => {
+  const { onevent } = socket;
+  /* eslint-disable-next-line no-param-reassign */
+  socket.onevent = function newOnevent(packet) {
+    const args = packet.data || [];
+    if (!onevent.call(this, packet)) {
+      /* eslint-disable-next-line no-param-reassign */
+      packet.data = ['default'].concat([args]);
```

Comment:
Any invocation of this function is because of an un-supported socket functionality in autolabcli. It is better to log an error message here. Please remember to update the tests accordingly.

Reply:
There are unfortunately no test cases which cover this part of the lines.